### DISCUSSION TITLE
Domotizzare Citofono Comelit 2638

### DISCUSSION THREAD

**Discussion by Valerio0188** [indomus/forum#2541] [GENERALE - Discussioni generiche]
Buongiorno a tutti,

spero che qualcuno sappia aiutarmi. Ho questo citofono Comelit 2638. Arrivano solo 2 fili collegati in L.
Il citofono da solo la possibilità di rispondere alla chiamata ed aprire il portone condominiale.
Dopo la pessima esperienza con il Ring Intercom che non ha mai funzionato bene, sto cercando di capire se è possibile smartizzare in qualche modo almeno l’apertura del portone, magari installando uno shelly, ma non so se tecnicamente è applicabile a questa situazione.

Qualcuno ha idea se è fattibile?

Grazie
![IMG_20230711_152921](https://github.com/indomus/forum/assets/135244280/209279f5-b593-4c93-b09b-609065e913e8)

**Comment by Valerio0188** [comment]
Nessuna idea?

**Reply by fabiosoft** [reply]
Ho appena comprato un Ring Intercom per poter domotizzare il Comelit 2638, che come te ha solo i cavi L collegati.
Ho collegato i (2) cavi B come da istruzioni, e lasciando gli altri scollegati, ma comunque non riesco ad aprire il portone d'ingresso... mi interessa quello maggiormente. A te funzionava?

**Reply by Valerio0188** [reply]
Ciao,
riuscivo ad aprire il portone (e non sempre) solo se veniva suonato il citofono. Non riuscivo ad aprirlo se il citofono non veniva suonato, cosa che invece per me era fondamentale.
Succedeva spesso, inoltre, di non ricevere nessuna notifica quando qualcuno suonava, insomma, a volte la ricevevo, a volte no, e non sono mai riuscito a capire il motivo.
Anche l'assistenza non è stata d'aiuto.
Alla fine l'ho reso.

**Reply by fabiosoft** [reply]
ci deve essere un modo smart di premere quella levettina... persino con uno switchbot non riesco a premere da fuori il pulsante...

**Reply by Valerio0188** [reply]
Se riesci a trovare il modo fammelo sapere  😅

**Reply by razorbac91** [reply]
Salderei due cavi sottili alle piazzole del pulsante cancello e li porto a un relè o un optoaccoppiatore.

**Reply by Valerio0188** [reply]
@razorbac91 nel mio caso non avrei modo di alimentare un relè, per questo una soluzione a batteria come il Ring Intercom poteva essere valida.

**Reply by matfil92** [reply]
Ma scusate usare uno Shelly 1 e saldare i contatti sul pulsante del citofono?

Chiedo perché io ho un 2708W/A con 2 cavi L-L e nessun'altra opzione per usare contatto pulito.

**Comment by ancientm74** [comment]
Ciao, ho il tuo stesso citofono, per aprire il cancello utilizzo uno shelly plus 1 (contatto pulito) collegato ai 2 connetori del citofono a contatto pulito, quelli dove hai la targhetta di carta per indenderci.

**Reply by fabiosoft** [reply]
unisci insieme quelli nella morsettiera? vicino a S+?

**Reply by garagara93** [reply]
Ciao, 

ho quasi lo stesso citofono, il 2608, con doppio portone, il primo che si apre solo dopo suonata ed il secondo classico.
Ho un accrocchio perfettamente funzionante che mi fa aprire il secondo portone: Switchbot con rialzo in legno su misura+esphome per BLEtoWIFI, perché la portata del BT non era sufficiente, cosi si apre in 3 secondi dal comando di home HA.

Non sono mai riuscito a risolvere l'apertura del primo portone però...
![photo_2023-11-23_20-15-38](https://github.com/indomus/forum/assets/42865915/0d8660e6-f78a-40de-9a58-760fbf530a25)
![photo_2023-11-23_20-15-36](https://github.com/indomus/forum/assets/42865915/2f782dd5-13f7-4fd8-9f85-1842becfbd75)

**Reply by Valerio0188** [reply]
> Ciao, ho il tuo stesso citofono, per aprire il cancello utilizzo uno shelly plus 1 (contatto pulito) collegato ai 2 connetori del citofono a contatto pulito, quelli dove hai la targhetta di carta per indenderci.

@ancientm74 come hai alimentato lo Shelly plus 1?

**Reply by garagara93** [reply]
Ma hai l'apertura con bussata? O anche senza?

**Reply by ancientm74** [reply]
Avendo sotto il citofono la cassetta con il passaggio dei cavi elettrici ho montano lo shelly li e passato i 2 cavi fino al citofono.

**Reply by Valerio0188** [reply]
> Ma hai l'apertura con bussata? O anche senza?

@garagara93 solo con bussata.


> Avendo sotto il citofono la cassetta con il passaggio dei cavi elettrici ho montano lo shelly li e passato i 2 cavi fino al citofono.

@ancientm74 purtroppo io non avrei possibilità di alimentare lo shelly perché non ho cassette vicino al citofono.
